src/components/DroughtHistory.vue

```diff
@@ -20,14 +20,14 @@
       <div id="chart-container">
         <img
           id="swarm_vertical"
-          src="@/assets/images/duration-chart/swarm_jd7d_2_western_compressed_vertical.png"
+          src="@/assets/images/duration-chart/swarm_jd7d_2pct_compressed_vertical.png"
           alt=""
         >
       </div>
       <div id="inset-container">
         <img
           id="inset-map"
```

Comment:
We have talked about changing these buttons to the 5 major droughts, which we are considering the main "events" to highlight. I'm going to leave as it is now, knowing that we're redesigning the timeline and that if there are 5 buttons rather than 11, that this still looks good as you first designed it.